frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/guest-agent-info/guest-agent-info-wrapper.ts

```diff
@@ -30,6 +30,8 @@ export class GuestAgentInfoWrapper extends Wrapper<
 
   getUserList = (): GuestOSUserWrapper[] => this.guestOSUserList;
 
+  getNumLoggedInUsers = (): number => this.guestOSUserList?.length;
```

Comment:
do we want number to be always returned? Or is returning undefined useful?

Reply:
Yeah... This was definitely a naive implementation on my part. I'll rewrite this part of the functionality.

Reply:
Done!